hal/src/stm32f2xx/bootloader_platform_10.c

```diff
@@ -1,37 +1,37 @@
 const unsigned char bootloader_platform_10_bin[] = {
```

Comment:
Totes, I probably tested against an older version so got the upgrade anyway. Although I did also see the endless loop when developing this initially. That's a bit worrying - anything we can to do help folks out of that if things do go wrong?